indexer/entity/src/projected_nft.rs

```diff
@@ -0,0 +1,55 @@
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Deserialize, Serialize)]
+#[sea_orm(table_name = "ProjectedNFT")]
+pub struct Model {
+    #[sea_orm(primary_key, column_type = "BigInteger")]
+    pub id: i64,
+    pub owner_address: Vec<u8>,
```

Comment:
Should this be a `Vec<u8>` instead of a foreign key into the stake credential table (careful: there is no guarantee this staking key exists in the table)

Reply:
> Should this be a `Vec<u8>` instead of a foreign key into the stake credential table (careful: there is no guarantee this staking key exists in the table)

as soon as there's no guarantee and this address is what we actually get from the plutus datum i think it makes sense to leave it like this